bigbluebutton/bigbluebutton
# feat(learning-dashboard): Learn more and feedback phrases and links

### What does this PR do?
- Add additional attributes to activities JSON
Creates a new field called 'other' inside the Meeting object, which is included in the activities JSON file. This field now includes two new metadata attributes: 'learning-dashboard-learn-more-link' and 'learning- dashboard-feedback-link'. These attributes define the URLs for the 'Learn more' and 'Feedback' anchor tags in the learning dashboard page. The URLs may vary depending on the customer/intitution, hence the need for metadata.
- Add 'learn more' and 'feedback' phrases and respective links
Adds two new phrase to the learning dashboard page. These phrases have embeded links that are defined by metadata: 'learning-dashboard-feedback -link' and 'learning-dashboard-learn-more-link'.


### How to test
Links should be defined with the new metadatas: 'learning-dashboard-learn-more-link' and 'learning-dashboard-feedback-link'.
- Create meating with new metadata setted
- Open learning dashboard
- See the new messages with links

### More
Work by @Arthurk12 

![image](https://github.com/user-attachments/assets/3b1a7f61-79db-411a-9c72-429bfa2fe66b)
